src/main/java/frc/robot/Constants.java

```diff
@@ -272,9 +282,12 @@ public static class ShootingConstants {
     public static final Measure<Time> SHOOT_TIME = Seconds.of(2.0);
     public static final Measure<Angle> AIM_TOLERANCE = Degrees.of(3.0);
 
-    public static final VelocityPitchInterpolator SHOOTER_INTERPOLATOR = new VelocityPitchInterpolator(List.of(
-      new ShootingSettings().distance(Meters.of(1)).velocity(RotationsPerSecond.of(40)).pitch(Rotations.of(0.1))
-    ));
+    public static final VelocityPitchInterpolator SHOOTER_PITCH_INTERPOLATOR = new VelocityPitchInterpolator(List.of(
+        new ShootingSettings().distance(Meters.of(1)).velocity(RotationsPerSecond.of(40)).pitch(Rotations.of(0.1))));
+
+    public static final VelocityPitchYawInterpolator SHOOTER_PITCH_YAW_INTERPOLATOR = new VelocityPitchYawInterpolator(
+        List.of(new ShootingSettingsYaw().distance(Meters.of(1)).velocity(RotationsPerSecond.of(40)).pitch(
+            Rotations.of(0.1)).yaw(Rotations.of(0.1))));
```

Comment:
The `VelocityPitchYawInterpolator` does pitch and yaw, that's its only reason for existing. There should only be one table.

Reply:
I made `VelocityPitchYawInterpolator` in order to be able to calculate Yaw, as horizontal aiming is controlled by the drivetrain in many places, however for auto I would like it to just be the Turret aiming. I'm pretty sure having that all in one table would make the calculations incorrect for many of your manual shooting commands, as they would take into account Yaw when they don't use it. If you do make `VelocityPitchInterpolator` calculate yaw at some point, and update the Commands to rotate the turret then we should merge them, however I feel it is best to keep them in seperate tables for now.

However I am probably missing something here, so feel free to correct me.

Reply:
See my other comment that I added while you were typing this.

I see that I misunderstood and didn't think deeply enough on this yesterday. My comment at the top of this thread is confused. `VelocityPitchInterpolator` is used to look up the velocity of the shooter wheels and the pitch of the turret. The yaw of the turret (and the drivetrain) should be defined by the location of the target (and optionally, the speed of the robot.) Yaw shouldn't be in a lookup table.